### DISCUSSION TITLE
[V5] How can I seed external json file data to database.

### DISCUSSION THREAD

**Discussion by Pratikkshirsagar** [adonisjs/core#1719] [Help]
Hi,
I wan to seed the various external json file data to database in a single go
1. how can I read the data of json file.
2. how can I export this data and consume in to seeder class.
3. how to run validator on it.

 Please help me out.

**Comment by thetutlage** [comment]
I believe it is simple as.

- Reading the JSON file and use `JSON.parse` to convert it to a Javascript object or array.
- Loop through it to add new rows using the Database provider or the Models (whichever is appropriate in your case)
- Use the validator (read docs) and validate rows before writing them to the database.
